LocalSettings.php

```diff
@@ -774,7 +774,7 @@
 		],
 	],
 	'wgCreateWikiCacheDirectory' => [
-		'default' => '/srv/mediawiki/w/cache'
+		'default' => '/srv/mediawiki/cache'
```

Comment:
1) Requires a puppet patch to create a writable directory
2) Please do not merge this _before_ denying access to this new folder: https://github.com/miraheze/puppet/blob/master/modules/mediawiki/templates/mediawiki-includes.conf.erb#L11-L13
3) Per the https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard, /var/cache may be the better location?

Reply:
1. yep that's why I marked this one as draft for now so it doesn't get merged before that's done.
2. Oh, right, thanks!
3. That actually may be better, yeah. Thanks!